pkg/webui/components/notification/index.js

```diff
@@ -71,15 +68,7 @@ const Notification = ({
               firstToUpper
               convertBackticks={Boolean(error)}
             />
-            {action && (
-              <Button
-                secondary
-                icon={buttonIcon}
-                onClick={action}
-                message={actionMessage}
-                type="button"
-              />
-            )}
```

Comment:
`action` is actually still used here `console/components/webhook-form/index.js`

Make sure you refactor that before merging. This should also mean that this crashes in Enterprise.

Reply:
Will resolve this for Enterprise in another PR